test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest>=6.2.5
```

Comment:
The idea is that we use `pytest` and `pytest-spark` for testing `soda-spark` projects. `pytest` is the commonly used library for python testing. And `pytest-spark` allows you to test your spark applications.